README.md

```diff
@@ -1,69 +1,76 @@
 # Visual Studio Code Kubernetes Tools
 [![Build Status](https://travis-ci.org/Azure/vscode-kubernetes-tools.svg?branch=master)](https://travis-ci.org/Azure/vscode-kubernetes-tools)
 
-A Visual Studio Code extension for interacting with Kubernetes clusters.  This extension combines
-the `vs-kubernetes` extension by @brendandburns and the `vs-helm` extension by @technosophos.
-
-## Configuring
-
-### Setting up your environment
-
-This extension assumes you have the following binaries on your `PATH`:
-
-   * `docker`
-   * `git`
-   * `kubectl` (optional)
-   * `helm` (optional)
-   * `draft` (optional)
-   * `minikube` (optional)
-
-For `kubectl`, `helm` and `draft` the binaries do not need to be on the system PATH. You can configure the extension by specifying the locations using the appropriate `vs-kubernetes -> vs-kubernetes.${tool}-path` configuration setting.  See [Extension Settings](#extension-settings) below.
-
-The extension can install `kubectl`, `helm` and `draft` for you if they are missing - choose **Install dependencies** when you see an error notification for the missing tool.  This will set `kubectl-path`, `helm-path` and `draft-path` entries in your configuration - the programs will *not* be installed on the system PATH, but this will be sufficient for them to work with the extension.
-
-If you are working with Azure Container Services or Azure Kubernetes Services, then you can install and configure `kubectl` using the `Kubernetes: Add Existing Cluster` command.
-
-If you plan to create managed clusters using Microsoft Azure (ACS or AKS), or to add clusters in those environments to your kubeconfig, then you will need Azure CLI 2.0.23 or above.  You do not need Azure CLI if you do not use Azure, or to interact with Azure clusters that are already in your kubeconfig.
-
-### Setting up your environment for Helm and Draft
-
-`helm` support requires that you have Helm installed and configured.
-
-To use the `Helm: Dry Run` command, your Kubernetes cluster must be running Tiller.
-
-For setting up `draft` you can provide a path to the binary via configuration (`vs-kubernetes.draft-path`) if it is not on your PATH.
-
-### Setting up the image repository path
+An extension for developers building applications to run in Kubernetes clusters,
+and for DevOps staff troubleshooting Kubernetes applications.  Features include:
+
+* View your clusters in an explorer tree view, and drill into workloads, services,
+  pods and nodes.
+* Browse Helm repos and install charts into your Kubernetes cluster.
+* Intellisense for Kubernetes resources and Helm charts and templates.
+* Edit Kubernetes resource manifests and apply them to your cluster.
+* Build and run Dockerfiles in your cluster.
```

Comment:
nit: maybe containers / container images might be a better wording for this?
(You don't necessarily run a Dockerfile)